### DISCUSSION TITLE
比赛时能固定一个跑法至育成结束吗？

### DISCUSSION THREAD

**Discussion by mtt-konan** [NateScarlet/auto-derby#300] [Q&A]
不想刷到逃的技能，虽然固定跑法会影响理论胜率，但是还是希望有类似插件能固定跑法，这样方便刷技能

**Comment by NateScarlet** [comment]
改 Race.style_score_v2 见 #158

**Reply by mtt-konan** [reply]
自己python没学好，不会写脚本改，于是直接改原文件race.py了。。效果是没问题的

![image](https://user-images.githubusercontent.com/102540842/165434291-22cf52a6-8e3e-476f-b97b-fd8fcd097b65.png)


 def style_scores(
        self,
        ctx: Context,
        *,
        _no_warn: bool = False,
    ) -> Tuple[float, float, float, float]:
        if not _no_warn:
            # TODO: remove old api at next major version
            import warnings
            warnings.warn(
                "use style_scores_v2 instead",
                DeprecationWarning,
            )
        #last = runing_style_score.compute(ctx, self, ctx.last, 1.1, 0.995, 0.9)
        #middle = runing_style_score.compute(ctx, self, ctx.middle, 1.0, 1, 0.95)
        #head = runing_style_score.compute(ctx, self, ctx.head, 0.8, 0.89, 1.0)
        #lead = runing_style_score.compute(ctx, self, ctx.lead, 0.5, 0.95, 1.1)
        last = 0
        middle = 0
        head = runing_style_score.compute(ctx, self, ctx.head, 0.8, 0.89, 1.0)
        lead = 0
        return last, middle, head, lead
